CommonData/ipcchannel.cpp

```diff
@@ -252,26 +252,14 @@ bool IPCChannel::jaspAlive()
 	if(!_isSlave)
 		return true; //if jasp asks its obviously alive
 
-	ifstream in;
-	in.open(_jaspHeartBeatPath);
-	if(!in.is_open()) {
-		Log::log() << "Could not find heartbeat file" << std::endl;
-		return false;
-	}
-
-	uint64_t stamp;
-	in >> stamp;
-	in.close();
-
+	uint64_t stamp = getFileModificationTime(_jaspHeartBeatPath);
```

Comment:
Nees this otherwise should include utils.h
```suggestion
	uint64_t stamp = Utils::getFileModificationTime(_jaspHeartBeatPath);
```